.golangci.yml

```diff
@@ -32,6 +37,10 @@ run:
     - web_src
 
 linters-settings:
+  stylecheck:
+    checks: ["all", "-ST1005", "-ST1003"]
+  nakedret:
+    max-func-lines: 0 
```

Comment:
```suggestion
    max-func-lines: 0 # currently disabled # TODO: fix it
```

Reply:
?

Reply:
Hm, I see I have expressed myself incorrectly above, with disabled I meant the linter has disabled the use of nakedreturns, not the linter itself is disabled.

With the setting on 0 it means that no lines may contain nakedreturns, so it'll trigger always when a naked return is used.